Updates to field-plot rmd and new tree rmd, OHDS project.

youngdjn opened the pull request:
Creating a PR from Victoria's branch...

"Small updates to Oren's code in field-plots (sorry I didn't know how github would display edits so not clear what I changed, see google doc notes). New standardization file for tree level data."

VictoriaMattsson committed:
Updates to field-plot rmd and new tree rmd, OHDS project.

Small updates to Oren's code in field-plots (sorry I didn't know how github would display edits so not clear what I changed, see google doc notes). New standardization file for tree level data.

youngdjn commented on contributor_data_to_OFO_schema/20250422_OHDS_Trees.Rmd:
```diff
@@ -0,0 +1,174 @@
+      Species.x == "ABAM" ~ "11",
+      Species.x == "ALRU2" ~ "351",
+      Species.x == "PISI" ~ "98",
+      Species.x == "UNCLS" ~ "0", # unclassified softwood
```
This should be `002`: https://www.fs.usda.gov/foresthealth/technology/pdfs/Appendix_F_Host.pdf

youngdjn commented on contributor_data_to_OFO_schema/20250422_OHDS_Trees.Rmd:
```diff
@@ -0,0 +1,174 @@
+      Species.x == "PREM" ~ "768",
+      Species.x == "ACCI" ~ "ACECIR", # from species-codes sheet
+      Species.x == "ACMA3" ~ "312",
+      Species.x == "TABR" ~ "231"), # check! species-codes says TABR2 = 231
```
This appears to be right, because according to USDA PLANTS, TABR is a forb, it's not that we're confusing it with another tree, they just took a shortcut and dropped the 2 like we do with CADE27.

youngdjn commented on contributor_data_to_OFO_schema/20250422_OHDS_Trees.Rmd:
```diff
@@ -0,0 +1,174 @@
+      DamA == "DT" ~ "90002", # dead top
+      DamA == "SB" ~ "9011", # open wound?
+      TRUE ~ ""),
+    damage_2 = "",
```
Can you confirm they don't have secondary damage codes etc?

VictoriaMattsson replied:
Yes, though there's a column for secondary and tertiary damage codes, none were populated.

youngdjn commented on contributor_data_to_OFO_schema/20250422_OHDS_Trees.Rmd:
```diff
@@ -0,0 +1,174 @@
+      TC == "13" ~ "L",
+      TC == "21" ~ "D",
+      TC == "22" ~ "D",
+      DamA == "DI" ~ "M"),  # I think this makes sense, but not sure
```
Nice, makes sense!

youngdjn commented on contributor_data_to_OFO_schema/20250422_OHDS_Trees.Rmd:
```diff
@@ -0,0 +1,174 @@
+      TC == "11" ~ "L",
+      TC == "12" ~ "L",
+      TC == "13" ~ "L",
+      TC == "21" ~ "D",
+      TC == "22" ~ "D",
```
Can you please add a comment here saying what 21 and 22 represent in the TC column?

VictoriaMattsson replied:
Will do! But now knowing we don't include downed trees, should I remove trees with TC = 22 (down dead)?

youngdjn replied:
Ah yes, they should be removed. Thanks.

VictoriaMattsson replied:
never mind, there actually weren't even any trees with that code

youngdjn reviewed (commented):
Looks good @VictoriaMattsson ! A few inline comments here, and I'll send a more involved comment by slack, mostly that I think there's a different stem coords csv to be using.

VictoriaMattsson committed:
Small updates according to Derek's suggestions!

youngdjn reviewed (commented):
Latest edits look good! I'd suggest that before merging, you make sure you can go all the way to the export and copy the values into the google sheet. In that process, you may discover some other things that need to be fixed. Once it's final and you've pasted the values in the google sheet, you should be good to merge.

youngdjn commented:
Latest edits look good @VictoriaMattsson ! I'd suggest that before merging, you make sure you can go all the way to the export and copy the values into the google sheet. In that process, you may discover some other things that need to be fixed. Once it's final and you've pasted the values in the google sheet, you should be good to merge.

VictoriaMattsson committed:
Small updates.

trees rmd: little exporting fix
plot rmd: small changes to forest type classifications
